Tiltfile

```diff
@@ -36,3 +38,6 @@ helm_crds, helm_non_crds = filter_yaml(
    kind = "^CustomResourceDefinition$",
 )
 k8s_yaml(helm_non_crds)
+
+k8s_yaml('docs/modules/opa/examples/getting_started/opa.yaml')
+k8s_resource(new_name='simple-opa', objects=['simple-opa:OpaCluster'], extra_pod_selectors=[{'app.kubernetes.io/name': 'opa'}], port_forwards=['8081:8081', '9476:9476'])
```

Comment:
Not sure about if this can be templated well, but I found it super convenient.

Reply:
Are you referring to the port forwards?

If so, maybe we can just assign fixed ports, like we already do for .... i dunno, something in templating I believe..Tilt itself maybe?

Reply:
Commented out in https://github.com/stackabletech/opa-operator/commit/8db6fc61a3cd7f493c78c03bfa3a71749467fd48#diff-c2ee8653e1d6b85f0aadf87cd438a9250806c052877248442be4d434cbc52425